server/flags-config/instant.schema.ts

```diff
@@ -1,50 +1,50 @@
-// instant-config
-// https://instantdb.com/dash?s=main&t=home&app=24a4d71b-7bb2-4630-9aee-01146af26239
+// http://localhost:3000/dash?s=main&t=home&app=24a4d71b-7bb2-4630-9aee-01146af26239
```

Comment:
(Looking here, I am going to just nix this; I don't think it's too helpful)

Reply:
It seems kind of useful to me! Just a little annoying for us since we use dev and prod.